src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -304,8 +330,10 @@ function showResponseCommentAddForm(recipientIndex, giverIndex, qnIndx) {
     $('#responseCommentAddForm' + id).focus();
 }
 
-function hideResponseCommentAddForm(recipientIndex, giverIndex, qnIndx) {
-    var id = '-' + recipientIndex + '-' + giverIndex + '-' + qnIndx;
+function hideResponseCommentAddForm(/* firstIndex, secondIndex, thirdIndex, fourthIndex */) {
+    var isIncludeSections = arguments.length === 4;
```

Comment:
I would prefer if the arguments are named. This way of accessing function parameters will make it very difficult for any future maintainer. For the `sectionIndex`, pass an object `opt` containing `sectionIndex` as a field. We should do it in this PR because doing this requires context of what the old arguments were, and a future developer taking this as a new issue would not have knowledge of this.